### PR TITLE
v6 documentation 

### PR DESCRIPTION
**Pull request by Faf4a**
# v6 documentation progress 

  

> ### Functions: 
- [x] [**_completed_**] util
- [x] [**_completed_**] info    
- [x] [**_completed_**] awaited
- [x] [**_completed_**] events
- [ ] [**_in-progress_** / **62%** ] misc
- [ ] [**_in-progress_** / **7%** ] calling

> ### Guides:
- [x] [**_completed_** #3 ]  sharding
- [x] [**_completed_** #7 ] music
- [x] [**_completed_** #6 ] panel
- [x] [**_completed_**] update home.md
- [x] [**_completed_**] command/variable handler
- [x] [**_completed_**] permissions / intents 
- [x] [**_completed_**] embed parser
- [x] [**_completed_**] hyperlink
- [x] [**_completed_**] variables
- [x] [**_completed_**] command properties / aliases etc
- [x] [**_completed_**] status
- [x] [**_completed_**] events
- [x] [**_completed_**] slash commands

### CODE REVIEW

**Comment by Faf4a** [comment]
_38 functions remaining_

**Comment by Faf4a** [comment]
_adding examples, command handler, status etc tmrw as well as the remaining 15 functions_

File: src/functions/info/serverDefaultMessageNotifications .md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+`$serverDefaultMessageNotifications ` will return given guild's default message notification type.
+
+## Usage
+
+```php
+$serverDefaultMessageNotifications[guild?]
+```
+
+## Parameters 
+
```
Comment (Leref) [inline src/functions/info/serverDefaultMessageNotifications .md:16]: There a blank space after `.md`

**Review by Leref — CHANGES_REQUESTED** [review]

File: src/functions/info/serverDefaultMessageNotifications .md (hunk 1 above)
Comment (Faf4a) [reply src/functions/info/serverDefaultMessageNotifications .md]: fixed

**Review by Leref — APPROVED** [review]

**Pull request merged**